woocommerce/woocommerce-android
# Track barcode format when the product search via SKU fails

Closes: #9126 


### Description


This PR tracks the barcode format when the product search via SKU fails. This information gives us which barcode format is failing the most. Do we need to handle the checksum for that format ...etc

More context: p1685078975281749-slack-C025A8VV728

### Testing instructions


1. Navigate to the order creation screen (Orders tab -> "+")
2. Tap on the barcode icon under the products section
3. Scan any invalid SKU with a different barcode format from the online barcode generator website
4. On product search via SKU failure, ensure product_search_via_sku_failure event is tracked
5. Ensure the `barcode_format` property is proper.
6. Repeat step 3 to 5 with different barcode formats.
